pynxtools/dataconverter/readers/json_map/reader.py

```diff
@@ -160,6 +160,14 @@ def read(self,
 
         ensure_all_required_fields_exist(template, new_template)
 
+        debugging = False
+        if debugging is True:
+            print("Reporting state of template before passing to HDF5 writing...")
+            for keyword, value in new_template.items():
+                print(f"{keyword}, {value}")
```

Comment:
Do we really need these kind of debug messages left? If we do need them, let's use the logging module so they can be correctly handled by executing code like Nomad running this instead of having print statements over an if statement.

Reply:
I see your point and agree for end users; given its set to False it will neither be seen nor have an effect upon execution. On the flipside none of these parsers are in yet a state where I would blindly start the verification for new developments and always first check that there is something in the template. As in few cases it works out of the box that code got removed and added in with each revision, yes might not be such a big deal to copy it in all the time, the better the parsers get the more I am becoming supportive for your point to remove this.